routes/baseActionsRouter.js

```diff
@@ -864,6 +864,16 @@ router.get("/about", (req, res) => {
 	res.render("about");
 });
 
+router.get("/richest-wallets", (req, res) => {
+	utils.getRichestWallets().then(function(results) {
+		global.richestWallets = results;
+
+		res.render("richest-wallets");
+
+		next();
```

Comment:
This will not work, in function definition you don't have next function. Remove this call.

Reply:
done